server/config/options.go

```diff
@@ -118,6 +121,7 @@ var DefaultConfig = Config{
 		FDBHardDrop: false,
 	},
 	Auth: AuthConfig{
+		EnableAuthCheck:  false,
```

Comment:
Can this be just Enabled?

Reply:
to be consistent with other packages configs.

Reply:
updated. thanks